extensions/amp-mustache/0.1/amp-mustache.js

```diff
@@ -33,14 +36,37 @@ export class AmpMustache extends AMP.BaseTemplate {
 
   /** @override */
   compileCallback() {
+    /** @private @const {!JsonObject} */
+    this.nestedTemplates_ = dict();
+    const shadowTemplate = this.element.cloneNode(true);
+    let index = 0;
+    iterateCursor(shadowTemplate.content.querySelectorAll('template'),
```

Comment:
In IE (and whichever browser doesn't had native templates) the `template.content` will be undefined. Consider just executing `querySelector` within the node itself as well.